src/lib/create-config.ts

```diff
@@ -182,17 +183,63 @@ const dateHandlingCallback = (connection, callback) => {
     });
 };
 
+const databaseSslFromFile = (caFilePath: string) => {
+    try {
+        return readFileSync(caFilePath).toJSON();
+    } catch (e) {
+        return {};
+    }
+};
+
+const filesExists = (...filePath: string[]): boolean => {
+    return filePath.every((file) => fs.existsSync(file));
+};
+
+const databaseSsl = () => {
+    if (process.env.DATABASE_SSL != null) {
+        return JSON.parse(process.env.DATABASE_SSL);
+    } else if (
+        process.env.DATABASE_SSL_CA_CONFIG != null &&
+        filesExists(process.env.DATABASE_SSL_CA_CONFIG)
```

Comment:
Not convinced about the file existence checks here. Is there a place where we want the file check to return true but return an empty object if it's unreadable or contains broken JSON?

Maybe it's the Python dev in me but I'd lean on just reading the thing and trapping the exception case

Reply:
I think that's a fair point, I think failing on non-existing files if you use the env vars is fair enough